tests/test_builders.py

```diff
@@ -4,16 +4,65 @@
 
 import pytest
 
-builder_map = ['cv', 'html', 'resume', 'publist', 'current-pending']
+from regolith.main import main
+
+builder_map = [
+    'cv',
+    'html',
+    'resume',
+    'publist',
+    'current-pending'
+]
 
 
 @pytest.mark.parametrize('bm', builder_map)
 def test_builder(bm, make_db):
     repo = make_db
     os.chdir(repo)
     if bm == 'html':
-        os.makedirs('templates/static')
-    subprocess.run(['regolith', 'build', bm, '--no-pdf'], check=True)
+        os.makedirs('templates/static', exist_ok=True)
+    subprocess.run(['regolith', 'build', bm,
+                    '--no-pdf'
+                    ], check=True)
+    os.chdir(os.path.join(repo, '_build', bm))
+    expected_base = os.path.join(os.path.dirname(__file__),
+                                 'outputs')
+    for root, dirs, files in os.walk('.'):
+        for file in files:
+
+            # Use this for bootstrapping the tests,
+            # confirm by hand that files look correct
+            # if root != '.':
+            #     os.makedirs(os.path.join(expected_base, bm, root),
+            #                 exist_ok=True)
+            # if root == '.':
+            #     shutil.copyfile(os.path.join(file),
+            #                     os.path.join(expected_base, bm, file))
+            # else:
+            #     shutil.copyfile(os.path.join(root, file),
+            #                     os.path.join(expected_base, bm, root, file))
```

Comment:
This commented block should be removed

Reply:
If you want a bootstrap function that is fine, but this is not a great way to handle that

Reply:
Any suggestions on how to handle the bootstrap?